### ISSUE TITLE
Better explanation of `execlineb` scripts

### ISSUE THREAD

**Issue by BrutalSimplicity** [just-containers/s6-overlay#432]
The documentation around `oneshot` scripts and `execline` scripts in general doesn't seem to mention that these scripts are not bash or shell scripts, but some s6-specific creation. It would be helpful to include this in the documentation in a way that stands out to the user. I was banging my head for a while trying to figure out what syntax of bash was being used.

This would be a _very_ helpful link to include in the documentation in the area discussing the `execline` script distinctions.
https://skarnet.org/software/execline/

**Comment by BrutalSimplicity** [comment]
Thank you for your response. I've got a few questions and thoughts on what you've said.

> That's why the documentation says that they should write their scripts as executables in a location of their choice, and only call these scripts in the up or down file. It has always been the intended usage of up and down files.

Where is this? There is a _lot_ in the documentation, and the sections are rather dense (for me) so I've probably missed this point.

An example where it's unclear to me when following along is the logging [section](https://github.com/just-containers/s6-overlay#logging). One of the use cases I'm exploring this tool for is the application of startup daemons for devcontainer environments. When I "followed" the instructions, I stubbornly ignored the `myapp-log-prepare/up` script's unusual syntax thinking it was some alternative format of bash. This led me down a bash syntax rabbit hole where I thought I'd missed out on some esoteric form of a bash `if` statement. Then finally having my google-fu defeated I relented to try what you had, and couldn't figure out why that worked until I finally dissected additional links in your instruction to stumble on the `execline` script format.

The bewildering snippet:
```
if { mkdir -p /var/log/myapp }
if { chown nobody:nogroup /var/log/myapp }
chmod 02755 /var/log/myapp
```

This is all to say that the examples generally follow well, but some of the more complex ones seem to make assumptions about prior knowledge of `s6` that may not be clear - _particularly for the impatient like myself_ 😅.
